usaspending_api/common/helpers/orm_helpers.py

```diff
@@ -26,8 +59,8 @@ class FiscalYear(Func):
 
 
 def generate_raw_quoted_query(queryset):
-    """
-    Generates the raw sql from a queryset with quotable types quoted.
+    """Generates the raw sql from a queryset with quotable types quoted.
```

Comment:
black?

Reply:
no

Reply:
Is it convention to start on the same line as the opening to the multi-line comment?

Reply:
As I recently learned: yes

Reply:
welp, if it's convention it's convention